index.html

```diff
@@ -199,5 +191,40 @@ <h1 class="about-title">About me</h1>
                 </div>
         </div>
     </section>
+    <section class="contact">
+        <div class="cont">
+            <h2 class="contact-text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+            <form action="https://formspree.io/f/xvoyeaqj" method="post">
+                <ul class="forms">
+                    <li>
+                        <input type="text" name="name" class="input" id="name" placeholder="Full name">
+                        <label for="name" class="label">Full name</label>
+                    </li>
+                    <li>
+                        <input type="email" name="email" class="input" id="email" placeholder="Email">
+                        <label for="email" class="label">Email</label>
+                    </li>
+                    <li> 
+                        <textarea class="area" rows="5" name="message">Enter text here</textarea>
+                    </li>
```

Comment:
- According to the project requirements, your Textarea input field should have a `maxlength=500`, please fix this 👍 👍